src/admin/components/card/order.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import { PieChart, Pie, Cell, Tooltip, ResponsiveContainer } from 'recharts';
+
+const Order = () => {
+  const data = [
+    { month: 'Jan', orders: 1 },
+    { month: 'Feb', orders: 2 },
+    { month: 'Mar', orders: 3 },
+    { month: 'Apr', orders: 4 },
+    { month: 'May', orders: 5 },
+    { month: 'Jun', orders: 6 },
+    { month: 'Jul', orders: 7 },
+    { month: 'Aug', orders: 8 },
+    { month: 'Sep', orders: 9 },
+    { month: 'Oct', orders: 10 },
+    { month: 'Nov', orders: 11 },
+    { month: 'Dec', orders: 12 },
+  ];
+
+  const COLORS = ['red', '#7E62D7', '#FF8C00', '#FFD700', '#36A2EB', '#FF6384'];
```

Comment:
Use theme file for these colors to keep it consistent

Reply:
done.